### DISCUSSION TITLE
do players that join a server automatically get audiolistener attached?

### DISCUSSION THREAD

**Discussion by ckdlskgkdrhd** [naverz/zepeto-studio-global#65] [Q&A]
when I play local via unity, I noticed how my spawned zepetocharacter doesn't have an audiolistener attached.
is this the case for the multiplayer mode as well.

**Comment by naverz-mkao** [comment]
Hi,

Yes that is the case!

No audio listener will be attached to player when you both play locally via unity and when you are in multiplayer mode.

The only time an audio listener will be automatically added is if you create a new Unity scene and pick the "Basic (Built-in) option.

![image](https://user-images.githubusercontent.com/107931355/187226352-96295eb7-f0b9-433a-884a-52cb2ec45170.png)

Otherwise, no audio listener will be added for you automatically.

**Comment by ckdlskgkdrhd** [comment]
thanks for the reply mkao :)
